src/modules/Login/Login.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {useFormik} from 'formik';
+import * as Yup from 'yup';
+import {TEMPLATEERRORSTRING} from '@Modules/Registration/Registration';
+import '../Registration/index.scss'
+
+
+export const Login = () => {
+  const formik = useFormik({
+    initialValues: {email: '', password: '',},
+    validationSchema: Yup.object({
+      email: Yup.string().email('Invalid email format').required(`${TEMPLATEERRORSTRING} email`),
+      password: Yup.string().required(`${TEMPLATEERRORSTRING} password`),
+    }),
```

Comment:
I would suggest to move this schema to object outside of the component, you can call it just `SCHEMA`.